frontend/src/components/PreviewReview/PreviewReview.tsx

```diff
@@ -63,14 +64,17 @@ const PreviewReview = ({
           mb="-12px"
           spacing={isPageView ? 20 : 0}
         >
-          <ChevronLeftIcon
-            boxSize="2em"
-            style={{
-              cursor: `${currentBook !== 0 ? "pointer" : "auto"}`,
-              color: `${currentBook !== 0 ? "black" : "lightgray"}`,
-            }}
-            onClick={handleLeftClick}
-          />
+          {!multiplebooks ? null : (
```

Comment:
For readability, we tend to prefer avoiding the not operator, `!`. So instead of doing `!multipleBooks  ?  null : (...code to show chevrons...)`, flip it around and do `multipleBooks ? (...code to show chevrons..) : null`